runtime/src/bank/partitioned_epoch_rewards/calculation.rs

```diff
@@ -1573,7 +2077,7 @@ mod tests {
         }
 
         let bank = Arc::new(Bank::new_for_tests(&genesis_config));
-        let slots_per_epoch = bank.epoch_schedule().slots_per_epoch;
+        let next_epoch_slot = bank.get_slots_in_epoch(bank.epoch);
```

Comment:
There are quite a few tests that don't account for the default epoch schedule using warmup slots. I made this change to account for said warmup.